mne_bids/tests/test_write.py

```diff
@@ -176,10 +191,10 @@ def test_kit():
     headshape_fname = op.join(data_path, 'test_hsp.txt')
     event_id = dict(cond=1)
 
-    raw = mne.io.read_raw_kit(
-        raw_fname, mrk=hpi_fname, elp=electrode_fname,
-        hsp=headshape_fname)
-    write_raw_bids(raw, bids_basename, output_path, events_data=events_fname,
+    raw = mne.io.read_raw_kit(raw_fname, mrk=hpi_fname, elp=electrode_fname,
+                              hsp=headshape_fname)
+    event_data = np.loadtxt(events_fname)  # we also accept nx3 ndarrays
+    write_raw_bids(raw, bids_basename, output_path, events_data=event_data,
```

Comment:
so using `np.loadtxt(events_fname)` (array) instead of `events_fname` (filename) in `write_raw_bids` directly showed us that there is an issue in `read_raw_bids` and how it handles `dtypes` in reading events files (see )

We need to fix that.

Interestingly, when I now revert to passing simply the `events_fname` to `write_raw_bids`, I get a different error: Namely, there is no `dig` information in the raw file :confused: 

```python
        dig = raw.info['dig']
        coords = dict()
>       fids = {d['ident']: d for d in dig if d['kind'] ==
                FIFF.FIFFV_POINT_CARDINAL}
E       TypeError: 'NoneType' object is not iterable

```

Reply:
... however, the `dig` info is certainly there if
1. performing the steps manually (ipython)
1. inserting an `assert False` right after `dig = raw.info['dig']` and then going into the code using the debugger:`pytest  mne_bids/tests/test_write.py::test_kit --pdb`